README.md

```
@@ -1,89 +1,114 @@
+- **Party-Wide Overview:** See every character's active projects, current progress, and time bank status in a single unified view.
+- **Manual Overrides:** GMs can manually adjust progress or targets for any character's project directly from this tab.
 
----
+ <div align="center">
+   <p>📸 <b>[Screenshot: Group Learning Tab]</b><br>
+   <i>The unified party overview on the Group actor sheet with the "Distribute Time" button.</i></p>
+ </div>
+ 
+ ---
+ 
+ ## 🎲 Training Mechanics
```

markdownlint-cli2 (0.22.0)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)